### ISSUE TITLE
Can I get propel to keep foreign key checks in MySQL by default?

### ISSUE THREAD

**Issue by kitsunde** [propelorm/Propel#1024]
I manually delete the FK disabling checks in all my migration because I don't want my database to be in an inconsistent state, I'd rather have the migration blow up. Can I make this the default behaviour?

**Comment by marcj** [comment]
What do you mean? If you specify foreign keys in your schema then you'll always get foreign key SQL statements in your migration files when they don't exist in the database. I actually don't understand what you exactly mean. Perhaps an example?

**Comment by kitsunde** [comment]
Whops wrong repo. This was for Propel2. The problem is this:
```sql
# This is a fix for InnoDB in MySQL >= 4.1.x
# It "suspends judgement" for fkey relationships until are tables are set.
SET FOREIGN_KEY_CHECKS = 0;
```
